Index: src/main/java/org/misha/contractor/messages/receive/CustomersMessageListener.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/main/java/org/misha/contractor/messages/receive/CustomersMessageListener.java	(revision )
+++ src/main/java/org/misha/contractor/messages/receive/CustomersMessageListener.java	(revision )
@@ -6,7 +6,7 @@
 import org.camunda.bpm.engine.delegate.DelegateExecution;
 import org.camunda.bpm.engine.delegate.JavaDelegate;
 import org.misha.Message;
-import org.misha.contractor.Adder;
+import org.misha.contractor.service.Adder;
 import org.misha.contractor.SumMessageContent;
 import org.misha.contractor.messages.send.impl.SumMessageSender;
 import org.misha.customer.TermsMessageContent;
